src/main/osd/osd_elements.c

```diff
@@ -1548,6 +1559,7 @@ static const uint8_t osdElementDisplayOrder[] = {
 #ifdef USE_PERSISTENT_STATS
     OSD_TOTAL_FLIGHTS,
 #endif
+    OSD_WARNING_LAND_NOW,
```

Comment:
Since this is separated from the actual warnings element would suggest `OSD_LAND_NOW_WARNING` instead.

Reply:
agree. Done